lidar.py

```diff
@@ -0,0 +1,86 @@
+import smbus
+import threading
+import time
+import RPi.GPIO as GPIO
+from collections import deque
+
+
+class LIDARLite(threading.Thread):
+
+    ADDR = 0x62
+    I2C_BUS_NO = 1
+    INTERRUPT_GPIO_PIN = 4
+
+    def __init__(self):
+        super().__init__()
+        self.bus = smbus.SMBus(LIDARLite.I2C_BUS_NO)
+        self.address = LIDARLite.ADDR
```

Comment:
why not use `self.ADDR` wherever we need this? is this intended to be configurable?

Reply:
Yes, in future we may want to add an optional argument, if we need multiple LIDARs with different addresses.